### PR TITLE
Add rubygem-foreman_rh_cloud package

### PR DESCRIPTION
**Pull request by ShimShtein**
This one is renaming `foreman_inventory_upload` package

### CODE REVIEW

File: packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.0.4
+Release: 1%{?foremandist}%{?dist}
+Summary: Connects Foreman with Red Hat Cloud services
+Group: Applications/Systems
+License: GPLv3
+URL: https://github.com/theforeman/foreman_rh_cloud
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Obsoletes: foreman_inventory_upload
```
Comment (evgeni) [inline packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec]:
```suggestion
Obsoletes: %{?scl_prefix}rubygem-foreman_inventory_upload
```

**Comment by evgeni** [comment]
I think you can safely remove the inventory_upload bits in the same PR too

**Comment by ShimShtein** [comment]
@evgeni, Done I think

File: comps/comps-foreman-rhel7.xml [hunk 2]
```diff
@@ -202,6 +202,7 @@
       <packagereq type="default">tfm-rubygem-fog-xenserver</packagereq>
       <packagereq type="default">tfm-rubygem-fog-xml</packagereq>
       <packagereq type="default">tfm-rubygem-font-awesome-sass</packagereq>
+      <packagereq type="default">tfm-rubygem-foreman_rh_cloud</packagereq>
```
Comment (ehelms) [inline comps/comps-foreman-rhel7.xml]: This will need to be in the katello comps file similar to how the inventory upload was

**Comment by ShimShtein** [comment]
@ehelms, moved to `comps-katello-server-rhel7.xml `

**Pull request merged**